spec.yaml

```diff
@@ -819,6 +911,7 @@ paths:
         ### Limitations
 
         - As of now, existing worker nodes cannot be modified.
+        - The number of availability zones can not be changed afterwards.
```

Comment:
The line above has `cannot`. Here we have `can not` now. Would be cool to align.

Reply:
If those things are bugging you. There is now a nice way how you can suggest changes. Marian did this already in this PR: https://github.com/giantswarm/api-spec/pull/117#discussion_r228089368